### PR TITLE
Md-sa2-16-21.Boris.Kydrevich.02.GIT.Local

### PR DESCRIPTION
**Pull request by SunBrooo**
My done homework.

### CODE REVIEW

File: Boris_Kydrevich/.gitkeep [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (pluhin) [inline Boris_Kydrevich/.gitkeep:1]: Please exclude this from PR

**Review by pluhin — CHANGES_REQUESTED** [review]

File: Boris_Kydrevich/.gitkeep (hunk 1 above)
Comment (SunBrooo) [reply Boris_Kydrevich/.gitkeep:1]: Sorry, i don't understand. This line or .gitkeep?

Comment (SunBrooo) [reply Boris_Kydrevich/.gitkeep:1]: > Please exclude this from PR

Sorry, i don't understand. This line or .gitkeep?

Comment (pluhin) [reply Boris_Kydrevich/.gitkeep:1]: File

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**